torch/_meta_registrations.py

```diff
@@ -87,6 +87,8 @@ def meta_fft_c2r(self, dim, normalization, lastdim):
 
 @register_meta(aten.copy_.default)
 def meta_copy_(self, src, non_blocking=False):
+    if src.device.type == "meta":
+        raise NotImplementedError("Cannot copy out")
```

Comment:
maybe go the extra mile and say "Cannot copy data from a meta tensor"?

Reply:
This is actually the current error message lol, and the test errors if I provide anything else.